site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
actually it's not an optimization. the `base` role is expected to be run just once (for instance, for configuring our base os image that we will ship) and this how i would expect it to be in production as well i.e. os image comes up configured with already run and then we use other roles to provision our services on top.